# 6.4 Hypothesis Tests of One Sample

## Objectives

- Compute probabilities using the Central Limit Theorem and demonstrate teh ability to interpret sampling distributions of both population proportions and means.
- Analyze a problem involving hypothesis testing, apply the correct techniques, and come to a conclusion for a claim about popultion proportion and mean, all this while using approprtiate levels of statistical significance, $p$-values, and determining what would constitute a type I and type II error.
- Analyze an application in the disciplines business, social sciences, psychology, life sciences, health science, and education, and utilize the correct statistical processes to arrive at a solution.

## Review and Additional Information

- In a **hypothesis test** problem, you may see words such as "the level of significance is 1%." The "1%" is the preconceived or preset $\alpha$.
- The statistician setting up the hypothesis test selects the value of $\alpha$ to use **before** collecting the sample data.
- When you calculate the $p$-value and draw the picture, the $p$-value is the area in the left tail, the right tail, or split evenly between the two tails. For this reason, we call the hypothesis test left-tailed, right-tailed, or two-tailed.
- **The alternative hypothesis,  $H_a$ , tells you if the test is left, right, or two-tailed.** It is the **key** to conducting the appropriate test.
- $H_a$ **never** has a symbol that contains an equal sign.
- **Thinking about the meaning of the $p$-value:** A data analyst (and anyone else) should have more confidence that he made the correct decision to reject the null hypothesis with a smaller $p$-value (for example, 0.001 as opposed to 0.04) even if using the 0.05 level for $\alpha$. Similarly, for a large $p$-value such as 0.4, as opposed to a $p$-value of 0.056 ($\alpha$ = 0.05 is less than either number), a data analyst should have more confidence that she made the correct decision in not rejecting the null hypothesis. This makes the data analyst use judgment rather than mindlessly applying rules.

The following examples illustrate a left-, right-, and two-tailed test.



***


### Example 4.1
$$\begin{align}
H_0:\ \mu = 5 \\
H_a:\ \mu < 5
\end{align}$$
Test of a single population mean. The less-than symbol on the alternative hypothesis $H_a$ tells us the test is left-tailed. The picture of the $p$-value is as follows:

<img src="lefttailed.jpeg" alt="Bell curve with area equal to p-value in the left tail.">

***


### Example 4.2
$$\begin{align}
H_0:&\ p \leq 0.2 \\
H_a:&\ p > 0.2
\end{align}$$

This is a test of a single population proportion. The greater-than symbol on the alternative hypothesis $H_a$ tells us the test is right-tailed. The picture of the $p$-value is as follows:

<img src="righttailed.jpeg" alt="A bell curve with area in the right tail shaded.">

***
### Example 4.3
$$\begin{align}
H_0:&\ p = 50 \\
H_a:&\ p \neq 50
\end{align}$$

This is a test of a single population proportion. The not-equal-to symbol on the alternative hypothesis $H_a$ tells us the test is two-tailed. In a two-tailed test, teach tail contains only half the $p$-value:

<img src="twotailed.jpeg" alt="A bell curve with equal regions in each tail shaded.">



***

The following table tells us which tailed test to use when:

| Symbol in $H_a$ | Which Tail |
|--|--|
| $<$ | Left-tailed |
| $>$ | Right-tailed |
| $\neq$ | Two-tailed |

## Steps and Examples

The general steps for completing a hypothesis test are

1. State the null and alternative hypotheses.
2. Assuming the null hypothesis is true, determine the features of the distribution of point estimates using the Central Limit Theorem.
3. Find the $p$-value of the point estimate.
4. Make a conclusion about the null hypothesis.

***


### Example 4.4
Jeffrey, as an eight-year old, established a mean time of 16.43 seconds for swimming the 25-yard freestyle, with a standard deviation of 0.8 seconds. His dad, Frank, thought that Jeffrey could swim the 25-yard freestyle faster using goggles. Frank bought Jeffrey a new pair of expensive goggles and timed Jeffrey for 15 25-yard freestyle swims and obtained the following swim times (in seconds):

14.96, 15.51, 15.54, 16.14, 15.55, 16.73, 16.4, 16.59, 14.76, 17.6, 17.68, 16.71, 14.87, 15.73, 16.42

Conduct a hypothesis test at the 5% significance level to conclude whether or not the goggles helped Jeffrey swim faster.

#### Solution
##### Step 1: State the null and alternative hypotheses.
Frank thinks Jeffrey's mean swim time would be faster with goggles; that is, $\mu < 16.43$. Since the symbol has no equal in it, this is the alternative hypothesis. Then we can write the null and alternative hypotheses as

$$\begin{align}
H_0:&\ \mu \geq 16.43 \\
H_a:&\ \mu < 16.43
\end{align}$$

##### Step 2: Assuming the null hypothesis is true, determine the features of the distribution of point estimates Using the Central Limit Theorem.
We are testing the population mean and we are told the population standard deviation. Then by the Central Limit Theorem, sample means are normally distributed with mean

$$ \mu_{\overline{X}} = \mu = 16.43 $$

and standard deviation

$$ \sigma_{\overline{X}} = \frac{\sigma}{\sqrt{n}} = \frac{0.8}{\sqrt{15}} = 0.2066. $$

##### Step 3: Find the  $p$-value of the point estimate.
The point estimate of the population mean is the sample mean $\bar{x}$.

In [2]:
x <- c(14.96, 15.51, 15.54, 16.14, 15.55, 16.73, 16.4, 16.59, 14.76, 17.6, 17.68, 16.71, 14.87, 15.73, 16.42)
n <- length(x)

xbar <- sum(x)/n
print(xbar)

[1] 16.07933


The sample mean is $\bar{x} = 16.0793$. The $z$-score associated with $\bar{x}$ is

$$ z = \frac{\bar{x} - \mu_{\overline{X}}}{\sigma_{\overline{X}}} = \frac{ 16.0793 - 16.43}{0.2066} = -1.6975. $$

Since the alternative hypothesis $H_a$ uses a less-than symbol, we will perform a left-tailed test. That is, the $p$-value is the probability $P(\bar{x} \leq 16.0793) = P(z \leq -1.6975)$. We use R to calculate this.

In [3]:
pnorm(q = -1.6975, lower.tail = TRUE)

[1] 0.04480109

So $P(\bar{x} \leq 16.0793) = P(z \leq -1.6975) = 0.0448$. Assuming the null hypothesis is true, that using goggles didn't improve Jeffrey's mean swim time, there is a 4.48% chance that a random sample of 15 of Jeffrey's swims with goggles would yield a sample mean of 16.0793 seconds or less.

##### Step 4: Make a conclusion about the null hypothesis.
The hypothesis test is at the 5% significance level, so $\alpha = 0.05$. Since

$$p\text{-value} = 0.0448 < 0.05 = \alpha, $$

we reject the null hypothesis. The chance of obtaining the sample mean we did if the null hypothesis were true is unlikely enough, we think it is more likely that the null hypothesis is not true.

We conclude that Jeffrey *does* improve his swim time using goggles.

***


### Example 4.5
A college football coach records the mean weight that his players can bench press as 275 pounds. Three of his players thought that the mean weight was more than that amount. They asked 30 of their teammates for their estimated maximum lift on the bench press exercise, obtaining the following data (in pounds):

205, 205, 205, 215, 215, 215, 225, 241, 241, 252, 252, 265, 265, 275, 275, 313, 313, 316, 316, 316, 316, 316, 338, 338, 341, 345, 345, 368, 368, 385

Conduct a hypothesis test using a 2.5% level of significance to determine if the bench press mean is more than 275 pounds.

#### Solution
##### Step 1: State the null and alternative hypotheses.
The three players think the mean bench press weight of the team is more than 275 pounds. Mathematically, we write this as $\mu > 275$. Since the greater-than symbol has no equal in it, this is our alternative hypothesis. Then the hypotheses are

$$\begin{align}
H_0:&\ \mu \leq 275 \\
H_a:&\ \mu > 275
\end{align}$$

##### Step 2: Assuming the null hypothesis is true, determine the features of the distribution of point estimates Using the Central Limit Theorem.

We are testing the population mean, but we are *not* told the population standard deviation. We will need to approximate the population standard deviation using the sample standard deviation and a $t$-distribution with degrees of freedom

$$df = n-1 = 30-1 = 29. $$

The mean of the distribution is

$$ \mu_{\overline{X}} = \mu = 275. $$

To find the standard deviation $\sigma_{\overline{X}} = \frac{s}{\sqrt{n}}$ of the distribution, we first need to find the standard deviation $s$ of the sample. To do so, first find the sample mean.

In [6]:
x <- c(205, 205, 205, 215, 215, 215, 225, 241, 241, 252, 252, 265, 265, 275, 275, 313, 313, 316, 316, 316, 316, 316, 338, 338, 341, 345, 345, 368, 368, 385)
n <- length(x)

xbar <- sum(x)/n
print(xbar)

[1] 286.1667


The sample mean is $\bar{x} = 286.1667$. Using this, we calculate the sample standard deviation.

In [8]:
s <- sqrt(sum( (x - xbar)^2 )/(n-1))
print(s)

[1] 55.89836


The sample standard deviation is $s = 55.8984$. Then the distribution standard deviation is

$$\sigma_{\overline{X}} = \frac{s}{\sqrt{n}} = \frac{55.8984}{\sqrt{30}} = 10.2056. $$

##### Step 3: Find the  $p$-value of the point estimate.
In step 2, we found that sample mean, which is the point estimate of the population mean, is $\bar{x} = 286.1667$. To find the $p$-value, we will need the $t$-score of $\bar{x}$:

$$t = \frac{\bar{x} - \mu_{\overline{X}}}{\sigma_{\overline{X}}} = \frac{286.1667 - 275}{10.2056} = 0.9962. $$

Since $H_a$ uses a greater-than symbol, we will perform a right-tailed test. So the $p$-value is $P(\bar{x} \geq 286.1667) = P(t \geq 0.9962)$. Let's use R to find this probability.

In [9]:
pt(q = 0.9962, df = 29, lower.tail = FALSE)

[1] 0.1636966

Then $P(\bar{x} \geq 286.1667) = P(t \geq 0.9962) = 0.1637$. That is, assuming the null hypothesis is true, that the team's mean lift weight is 275 pounds, there is a 16.37% chance that if we randomly sample 30 team members, their mean lift weight would be at least 286.1667 pounds.

##### Step 4: Make a conclusion about the null hypothesis.
The level of significance is 2.5%, so $\alpha = 0.025$. Since

$$ p\text{-value} = 0.1637 \geq 0.025 = \alpha, $$

we do not reject the null hypothesis.

The evidence is not strong enough to conclude that the team mean lift weight is greater than 275 pounds.

***


### Example 4.6
Joon believes that 50% of first-time brides in the United States are younger than their grooms. She performs a hypothesis test to determine if the percentage is the same or different from 50%. Joon samples 95 first-time brides and 51 reply that they are younger than their grooms. For the hypothesis test, she uses a 1% level of significance.

#### Solution
##### Step 1: State the null and alternative hypotheses.
Joon wants to know if the percent of first-time brides that are younger than their grooms is 50% (that is, if $p = 0.50$) or different from 50% (that is, if $p \neq 0.50$). Then the hypotheses are

$$\begin{align}
H_0:&\ p = 0.50 \\
H_a:&\ p \neq 0.50
\end{align}$$

##### Step 2: Assuming the null hypothesis is true, determine the features of the distribution of point estimates Using the Central Limit Theorem.

We are testing the population proportion. By the Central Limit Theorem, sample proportions are normally distributed with mean

$$ \mu_{P'} = p = 0.50 $$

and standard deviation

$$ \sigma_{P'} = \sqrt{\frac{p(1 - p)}{n}} = \sqrt{\frac{0.50(1 - 0.50)}{95}} = 0.0513. $$

##### Step 3: Find the  $p$-value of the point estimate.
The point estimate of the population proportion is the sample proportion

$$ p' = \frac{x}{n} = \frac{51}{95} = 0.5368. $$

Using the features of the distribution determined in step 2, we can calculate the $z$-score of the point estimate $p'$:

$$ z = \frac{p' - \mu_{P'}}{\sigma_{P'}} = \frac{0.5368 - 0.50}{0.0513} = 0.7173. $$

Since $H_a$ uses a not-equal-to symbol, we will perform a two-tailed test. That means half of the $p$-value is in each tail. We will first calculate the half of the $p$-value in the right tail, as represented by $P(p' \geq 0.5368) = P(z \geq 0.7173)$. (We find the half of the $p$-value in the right tail since our point estimate $p' = 0.5368$ is to the right of $p = 0.50$.)

In [11]:
pnorm(q = 0.7173, lower.tail = FALSE)

[1] 0.2365945

So $P(p' \geq 0.5368) = P(z \geq 0.7173) = 0.2366.$ But remember, this is only half the $p$-value. that means the full $p$-value is

$$ p\text{-value} = 2(0.2366) = 0.4732. $$

So, assuming the null hypothesis that the proportion of first-time brides younger than their grooms *is* 50%, there is a 47.32% chance that a random survey would yield a sample proportion at least as extreme as $p' = 0.5368 = 53.68\%$.

##### Step 4: Make a conclusion about the null hypothesis.
The level of significance of the hypothesis test is 1%, so $\alpha = 0.01$. Since

$$ p\text{-value} = 0.4732 \geq 0.01 = \alpha, $$

we cannot reject the null hypothesis.

The evidence is insufficient to conclude that the proportion of first-time brides that are younger than their grooms is different than 50%.


***

### Example 4.7

In [ ]:
#**VID=Kiw1wDJ_goY**#

***

### Example 4.8

In [1]:
#**VID=MEAUDl8wjN4**#

***

### Example 4.9

In [ ]:
#**VID=_7DjYMmR7eE**#

***

## Attribution ##
Significant portions of this textbook were written by Barbara Illowsky and Susan Dean in their textbook *Introductory Statistics*, published by OpenStax.

Access for free at [https://openstax.org/books/introductory-statistics/pages/1-introduction](https://openstax.org/books/introductory-statistics/pages/1-introduction)